src/main/kotlin/no/nav/syfo/papirsykmelding/tilsyfoservice/SendTilSyfoserviceService.kt

```diff
@@ -9,39 +9,38 @@ import no.nav.syfo.papirsykmelding.tilsyfoservice.kafka.SykmeldingSyfoserviceKaf
 import no.nav.syfo.papirsykmelding.tilsyfoservice.kafka.model.KafkaMessageMetadata
 import no.nav.syfo.papirsykmelding.tilsyfoservice.kafka.model.SykmeldingSyfoserviceKafkaMessage
 import no.nav.syfo.papirsykmelding.tilsyfoservice.kafka.model.Tilleggsdata
-import no.nav.syfo.persistering.db.postgres.hentSykmelding
+import no.nav.syfo.persistering.db.postgres.hentSykmeldingMedId
 import no.nav.syfo.utils.extractHelseOpplysningerArbeidsuforhet
 
 class SendTilSyfoserviceService(
     private val sykmeldingSyfoserviceKafkaProducer: SykmeldingSyfoserviceKafkaProducer,
     private val databasePostgres: DatabasePostgres
 ) {
 
-    val mottakId = ""
-
-    fun start() {
-        val sykmelding = databasePostgres.connection.hentSykmelding(mottakId)
+    fun sendTilSyfoservice(sykmeldingId: String) {
+        val sykmelding = databasePostgres.connection.hentSykmeldingMedId(sykmeldingId)
         if (sykmelding != null) {
-            log.info("sender sykmelding med mottaksid {} til syfoservice", mottakId)
+            log.info("sender sykmelding med sykmeldingId {} til syfoservice", sykmeldingId)
```

Comment:
Merk at det vi egentlig tar inn og bruker mot databasen er mottakId, ikke sykmeldingsid. For papirsykmeldinger er disse to id-ene tilfeldigvis like, for nå.. 😬

Reply:
Hmm, er det? Jeg trodde jeg endret det til sykmeldingid. Skal vi støtte flere mottakskanaler enn API-et?

Reply:
Ah, ser nå at det er en ny metode som kalles :) Men, returnerer den metoden det komplette sykmeldingsdokumentet? Mener det var en grunn til at jeg ikke gadd å bytte metode og heller bare brukte den som tok inn mottaksid 😅 

Jeg vet ikke hva du tenker på med mottakskanaler, men det er kanskje ikke så relevant her..? :)

Reply:
Hehe, okay. Nei, jeg stusset bare på at du sa at vi tok imot mottakid, så jeg lurte egentlig bare på om jeg hadde oversett et annet api eller en annen kafka-kø eller noe

Reply:
Jeg er usikker på om metoden returnerer hele sykmeldingsdokumentet. Jeg får grave litt og se :)

Reply:
Med unntak av WHERE-klausulen ser det ut til å være samme spørring og mapping som gjøres i for hentSykmeldingMedId(), så jeg er ganske trygg på at utfallet burde bli det samme som før (med forbehold om sort magi, naturligvis).

Men jeg ser det er flere hentSykmelding-metoder som gjør... diverse. Det kan hende du prøvde deg på noen av dem forrige gang? :-)

Reply:
Ja, det er jammen ikke godt å si når det er så mye som ligner 😅